girlpower/06_Loading_Geometry_Overview.v4p

```diff
@@ -584,6 +562,35 @@
    </NODE>
    <LINK srcnodeid="251" srcpinname="Is Valid" dstnodeid="282" dstpinname="Y Input Value">
    </LINK>
-   <PACK Name="dx11" Version="33.0">
-   </PACK>
+   
```

Comment:
Updating path like that is not too good, but seems on previous release 4v tries to resolve system name and not path anyway, so it's ok.